tests/fake_models.py

```diff
@@ -111,7 +126,12 @@ class FakeBaseModelBoringWithAnnotations(object):
     .. no_pii:: No PII.
     """
 
-    _meta = MagicMock(app_label='fake_app_3', object_name='FakeBaseModelBoring', abstract=False, proxy=False)
+    _meta = MagicMock(
+        app_label='fake_app_3',
+        object_name='FakeBaseModelBoringWithAnnotations',
+        abstract=False,
+        proxy=False
```

Comment:
While debugging an issue with `requires_annotations` it was unclear what those values might be, I figured it would make future debugging easier to have them explicit.